### DISCUSSION TITLE
Make obsidian plugins work in foam.

### DISCUSSION THREAD

**Discussion by thoka** [foambubble/foam#1058] [General]
### Is your feature request related to a problem? Please describe.

I like obsidian for its great community plugin repository. 
What would have to be done to make these plugins work in foam to?

### Describe the solution you'd like

I would like to se some discussion evolve about the possibility to use obsidian plugins in foam.

### Describe alternatives you've considered

One can use obsidian and foam side by side on the same vault [as described by some redditors](https://www.reddit.com/r/ObsidianMD/comments/sje8by/can_i_work_on_my_vault_with_both_obsidian_and/).

**Comment by riccardoferretti** [comment]
I like the idea, I don't know how practical it is though as I am not familiar enough with the Obsidian API and plugins.
If someone is interested in diving deeper, please do

**Comment by domportera** [comment]
super psyched that this project exists, I just found out about it

but as an obsidian user who makes heavy use of their extensions, I would need this to migrate

**Comment by Lloyd-Jackman-UKPL** [comment]
I am actually looking at a migration from Obsidian to Foam and am best part there if I'm honest.  Might be interesting to rather see if there are specific extensions of interest that don't have equivalence in the VS Code marketplace and make a judgement call whether it is easier to port Obsidian apps, build on what is already in the VS Code marketplace or build anew.
